### PR TITLE
fix(entity): configure jib to synchronize launch of entity service an…

### PR DESCRIPTION
**Pull request by HoucemKacem**

### CODE REVIEW

File: entity-service/build.gradle.kts [hunk 1]
```diff
@@ -34,6 +36,9 @@ tasks.bootRun {
 
 jib.from.image = project.ext["jibFromImage"].toString()
 jib.to.image = "stellio/stellio-entity-service"
-jib.container.jvmFlags = listOf(project.ext["jibContainerJvmFlag"].toString())
+jib.container.entrypoint = listOf("/bin/sh")
+jib.container.args = listOf("-c", "chmod +x /database/wait-for-neo4j.sh && " +
+    "/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- echo \"Connected to Neo4j\" && " +
```
Comment (bobeal) [inline entity-service/build.gradle.kts:41]: why don't you do `wait-for-neo4j.sh ... -- java ...`?

Comment (bobeal) [reply entity-service/build.gradle.kts:41]: it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?

Comment (HoucemKacem) [reply entity-service/build.gradle.kts:41]: > it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?

Yes

Comment (HoucemKacem) [reply entity-service/build.gradle.kts:41]: > why don't you do `wait-for-neo4j.sh ... -- java ...`?

You mean ?
```
"/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- echo \"Connected to Neo4j\" -- " +
    "java " + project.ext["jibContainerJvmFlag"].toString() + " -cp /app/resources:/app/classes:/app/libs/* " + mainClass)
```

Comment (HoucemKacem) [reply entity-service/build.gradle.kts:41]: Or ?

```
"/database/wait-for-neo4j.sh neo4j:7687 -t 40 -- " +
    "java " + project.ext["jibContainerJvmFlag"].toString() + " -cp /app/resources:/app/classes:/app/libs/* " + mainClass
```

Comment (bobeal) [reply entity-service/build.gradle.kts:41]: the second one

Comment (bobeal) [reply entity-service/build.gradle.kts:41]: > > it is not possible to add the script with the correct permissions by using `extraDirectories.permissions`?
> 
> Yes

why it is not possible?

Comment (HoucemKacem) [reply entity-service/build.gradle.kts]: sorry i meant it's possible

File: entity-service/build.gradle.kts [hunk 2]
```diff
@@ -34,6 +36,9 @@ tasks.bootRun {
 
 jib.from.image = project.ext["jibFromImage"].toString()
 jib.to.image = "stellio/stellio-entity-service"
-jib.container.jvmFlags = listOf(project.ext["jibContainerJvmFlag"].toString())
+jib.container.entrypoint = listOf("/bin/sh")
```
Comment (bobeal) [inline entity-service/build.gradle.kts:39]: and last point, why splitting it in entrypoint and args and not putting all in entrypoint ?

**Pull request merged**